{{cookiecutter.repo_name}}/Makefile

```diff
@@ -2,20 +2,52 @@
 
 .PHONY: dev.clean dev.build dev.run
 
+REPO_NAME := {{cookiecutter.repo_name}}
+PACKAGE_NAME := {{cookiecutter.package_name}}
+SETTINGS_FILE := $(PACKAGE_NAME).locale.settings
```

Comment:
This should break since Django will try to import `locale` and that will cause confusion. AFAIK it will only work if it's moved to `$(PACKAGE_NAME).settings`. The actual file should be renamed as well.

Reply:
Yeah, this just works with the xblock installed to the virtualenv. I'm moving back to the `manage.py` to get rid of this confusion.